docs/changelog.md

```diff
@@ -0,0 +1,68 @@
+## Changelog
+
+### v0.7.0(31/12/2020)
+
+- Add more evaluation metrics
+- Fix bugs
+
+#### New Features
+
+- Remove installation of MMCV from requirements. (#90)
+- Add more evaluation metrics: precision, recall and F-1 score. (#93)
+- Allow overriding configs through input argument when testing and inferencing. (#91 & #96)
```

Comment:
It may be better that "Allow config override during testing and inference by running "test.py" with `--options`"